modules/effects/src/provide_effects.ts

```diff
@@ -66,6 +69,13 @@ export function provideEffects(
 
   return makeEnvironmentProviders([
     effectsClasses,
+    EffectSources,
```

Comment:
I think this will cause an issue when `provideEffects` is used in a child route, but wanted to open the discussion for the fix.

Reply:
Yes, that would be a problem.